src/test/java/com/google/api/codegen/protoannotations/testdata/php_library.baseline

```diff
@@ -2628,7 +2628,8 @@ class LibraryServiceGapicClient
      * @param string $archive
      * @param string $book
      * @return string The formatted book_from_archive resource.
-     * @experimental
+     * @deprecated Multi-pattern resource names will have unified formatting functions.
```

Comment:
@dwsupplee these two resource names were introduced in [a previous PR](https://github.com/googleapis/gapic-generator/pull/3016/files#diff-fc97e801b7222b7481527c795281e442R64) (Sorry for the inconvenience of not showing the changes in the same PR)